CHANGELOG.md

```diff
@@ -13,7 +13,13 @@
 * Increment vCD API version used from 27.0 to 29.0
     * Remove fields `VdcEnabled`, `VAppParentHREF`, `VAppParentName`, `HighestSupportedVersion`, `VmToolsVersion`, `TaskHREF`, `TaskStatusName`, `TaskDetails`, `TaskStatus` from `QueryResultVMRecordType`
     * Add fields `ID, Type, ContainerName, ContainerID, OwnerName, Owner, NetworkHref, IpAddress, CatalogName, VmToolsStatus, GcStatus, AutoUndeployDate, AutoDeleteDate, AutoUndeployNotified, AutoDeleteNotified, Link, MetaData` to `QueryResultVMRecordType`, `DistributedInterface` to `NetworkConfiguration` and `RegenerateBiosUuid` to `VMGeneralParams`
-    * Change to pointers `DistributedRoutingEnabled` in `GatewayConfiguration` and `DistributedInterface` in `NetworkConfiguration`
+    * Change to pointers `DistributedRoutingEnabled` in `GatewayConfiguration` and
+    `DistributedInterface` in `NetworkConfiguration`
+* Add new fields to type `GatewayConfiguration`: `FipsModeEnabled` -
```

Comment:
Says `fields` in plural, but only a single field is referenced.

Reply:
Indeed. Fixed.